### PR TITLE
Fixes issue #83

### PR DESCRIPTION
**Pull request by saurabhjn76**
Fixes issue #83

Changes: 

- added FileProvider tag in AndroidManifest.xml to give access to the particular [link](http://stackoverflow.com/questions/38200282/android-os-fileuriexposedexception-file-storage-emulated-0-test-txt-exposed)

- implemented share intent 

Screenshots for the change: 
whatsapp image 2017-04-06 at 12 36 57 am 1
whatsapp image 2017-04-06 at 12 36 57 am

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @saurabhjn76 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/saurabhjn76/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/gallery/PhimpMeGallery.java [hunk 1]
```diff
@@ -304,17 +306,22 @@ public void onClick(DialogInterface dialog, int which) {
...
-                shareIntent.putExtra("image-path", filePath.get(position));
-                shareIntent.putExtra("aspectX", 0);
-                shareIntent.putExtra("aspectY", 0);
-                shareIntent.putExtra("scale", true);
-                shareIntent.putExtra("activityName", "PhimpMeGallery");
-                startActivityForResult(shareIntent, 1);
 
-                Log.d("Pawan", "Share");
+                startActivityForResult(shareIntent, 1);
+                Log.d("Pawan", "Share");*/
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/gallery/PhimpMeGallery.java]: Please remove unwanted code.

File: app/src/main/res/xml/provider_paths.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <external-path name="external_files" path="."/>
+</paths>
```
Comment (Abhi2424shek) [inline app/src/main/res/xml/provider_paths.xml]: Add a new line at eof

**Comment by mariobehling** [comment]
Good job. We can iterate from here. Thanks!

**Pull request merged**